mod-data-export/src/test/java/org/folio/utils/MarcFileData.java

```diff
@@ -0,0 +1,483 @@
+package org.folio.utils;
+
+import java.io.ByteArrayInputStream;
+import java.util.Comparator;
+import org.marc4j.MarcReader;
+import org.marc4j.MarcStreamReader;
+import org.marc4j.marc.DataField;
+import org.marc4j.marc.MarcFactory;
+import org.marc4j.marc.Record;
+import org.marc4j.marc.impl.MarcFactoryImpl;
+import org.marc4j.marc.impl.SubfieldImpl;
+
+public class MarcFileData {
+
+    private static final char EMPTY_CHAR = ' ';
+
+    public static Record convertByteArrayToRecordSortedByTag(byte[] marcFile) {
+        ByteArrayInputStream inputStream = new ByteArrayInputStream(marcFile);
+        MarcReader reader = new MarcStreamReader(inputStream);
+        Record record = reader.next();
+        record.getDataFields().sort(Comparator.comparing(DataField::getTag));
+        return record;
+    }
+
+    public static Record createTestRecordSortedByTag() {
+        MarcFactory marcFactory = new MarcFactoryImpl();
+        Record record = marcFactory.newRecord();
+
+        //0
+        DataField dataField = marcFactory.newDataField();
```

Comment:
Hi @AntonAntonich , I believe it is better to split checking fields to separate methods - it will be easier to find incorrect behavior if something will go wrong, karate report will be more informative.